packages/flutter/lib/src/scheduler/ticker.dart

```diff
@@ -431,17 +429,17 @@ class TickerFuture implements Future<void> {
   }
 
   @override
-  Future<void> catchError(Function onError, { bool test(dynamic error) }) {
+  Future<void> catchError(Function onError, { bool Function(Object)? test }) {
     return _primaryCompleter.future.catchError(onError, test: test);
   }
 
   @override
-  Future<R> then<R>(FutureOr<R> onValue(void value), { Function onError }) {
+  Future<R> then<R>(FutureOr<R> onValue(void value), { Function? onError }) {
     return _primaryCompleter.future.then<R>(onValue, onError: onError);
   }
 
   @override
-  Future<void> timeout(Duration timeLimit, { dynamic onTimeout() }) {
+  Future<void> timeout(Duration timeLimit, {FutureOr<void> onTimeout()?}) {
```

Comment:
Since you changed it above: Change this one also to the new function syntax?

Reply:
Done